zilliqa/tests/it/main.rs

```diff
@@ -206,21 +217,23 @@ impl Network {
         // Pause time so we can control it.
         zilliqa::time::pause_at_epoch();
 
-        for node in &nodes[1..] {
```

Comment:
nodes will now try to join when they see proposed blocks they are not in the committee for